database/seeds/DatabaseSeeder.php

```diff
@@ -11,6 +11,12 @@ class DatabaseSeeder extends Seeder
      */
     public function run()
     {
-        // $this->call(UsersTableSeeder::class);
+
+        \Illuminate\Support\Facades\DB::statement('SET FOREIGN_KEY_CHECKS=0');
+
+        $this->call(PostStatusesTableSeeder::class);
+        $this->call(PostsTableSeeder::class);
+        $this->call(UsersTableSeeder::class);
+
```

Comment:
Setup phpcs for PSR-2 standard and mark phpcs notifications as `errors` it will help you to keep your code clean.
After that check all your files for phpcs warnings.

<img width="481" alt="code-review" src="https://user-images.githubusercontent.com/18187060/70279934-1adbe600-17c0-11ea-8e0c-2ec941c8af35.png">

<img width="458" alt="image" src="https://user-images.githubusercontent.com/18187060/70280102-668e8f80-17c0-11ea-8861-a55c6927c237.png">

Reply:
Fixed.

Have question about namespacing for migrations.